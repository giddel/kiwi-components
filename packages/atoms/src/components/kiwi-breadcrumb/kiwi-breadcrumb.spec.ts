import { newSpecPage } from '@stencil/core/testing';
import { KiwiBreadcrumb } from './kiwi-breadcrumb';

describe('kiwi-breadcrumb', () => {
  it('renders', async () => {
    const page = await newSpecPage({
      components: [KiwiBreadcrumb],
      html: `<kiwi-breadcrumb></kiwi-breadcrumb>`,
    });
    expect(page.root).toMatchSnapshot();
  });
});
