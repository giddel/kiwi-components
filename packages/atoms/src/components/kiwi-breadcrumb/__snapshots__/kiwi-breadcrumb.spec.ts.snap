// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`kiwi-breadcrumb renders 1`] = `
<kiwi-breadcrumb>
  <!---->
  <nav>
    <ol class="align-items-center breadcrumb d-flex"></ol>
  </nav>
</kiwi-breadcrumb>
`;
